python/erdos/__init__.py

```diff
@@ -83,6 +85,14 @@ def connect(
     node_id = _num_py_operators
     logger.debug("Connecting operator #{num} ({name}) to the graph.".format(
         num=node_id, name=config.name))
+    if config._addr != None:
+        global node_addrs
+        global num_procs
```

Comment:
Rename `num_procs` to `processes_per_address`, and compute as follows:

```suggestion
        addresses = list(node_to_address.values())
        processes_per_address = {a: addresses.count(a) for a in set(addresses)}
```